src/styles/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$primarycolor: #060b35;
+$secondarycolor: #616070;
+$buttoncolor: #fff;
+$hoverbuttoncolor: #00acdc;
+$backgroundbuttoncolor: #00acdc;
+$bordercolor: #00acdc;
```

Comment:
the correct naming convention would be `primary-color` `secondary-color` and so on.